### PR TITLE
PLANNER-1606 Remove all references to webexamples from the website

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

File: download/releaseNotes/releaseNotes7.adoc [hunk 1]
```diff
@@ -27,6 +27,11 @@ like this:
 
 For more information, please refer to link:../../learn/documentation.html[Documentation].
 
+=== OptaPlanner Web Examples are no more
+
+The optaplanner-webexamples module was removed from distribution and the Git repository. It has been superseded by the
+link:../../learn/rhpds/[OptaWeb examples].
+
```
Comment (triceo) [inline download/releaseNotes/releaseNotes7.adoc]: This is gonna need changing, as master has just been upgraded to 7.28.

File: download/releaseNotes/releaseNotes7.adoc [hunk 2]
```diff
@@ -27,6 +27,11 @@ like this:
 
 For more information, please refer to link:../../learn/documentation.html[Documentation].
 
+=== OptaPlanner Web Examples are no more
+
+The optaplanner-webexamples module was removed from distribution and the Git repository. It has been superseded by the
+link:../../learn/rhpds/[OptaWeb examples].
```
Comment (ge0ffrey) [inline download/releaseNotes/releaseNotes7.adoc]: rhdps -> optaweb github link

**Comment by ge0ffrey** [comment]
In optaplanner-website, there is this file:
  learn/slides/optaplanner-presentation/Development/methodologyOverview.png
which is a copy of the png file (not the svg file) of optaplanner repo. Would you mind updating that too?
(I know DRY - ways to fix this are welcome, but it's not that simple)

**Comment by triceo** [comment]
@ge0ffrey Updated.

**Pull request merged**